test/constant.ts

```diff
@@ -18,3 +18,9 @@ export const KIBANA_SERVER_PASSWORD: string = 'kibanaserver';
 
```

Comment:
same here for password.

Reply:
the default user name and password for ODFE are both `kibanaserver`